mysql/scale-mysql-server/scale-mysql-server.sh

```diff
@@ -28,13 +28,20 @@ az monitor metrics list \
 --metric storage_used \
```

Comment:
__Required__: az monitor metrics list fails:

```
ERROR: az monitor metrics list: error: the following arguments are required: --resource
```

Scripts are required to be tested prior to submission. https://review.docs.microsoft.com/en-us/help/contribute/contribute-scripts-pr-process?branch=master#test-script-changes